src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -362,10 +362,15 @@ bool LogConfiguration::parse_command_line_arguments(const char* opts) {
       *next = '\0';
       str = next + 1;
     } else {
+      str = NULL;
       break;
     }
   }
 
+  if (str != NULL) {
+    log_warning(logging)("Options are specified more than the max number in -Xlog.");
```

Comment:
Might I suggest:
`log_warning(logging)("Ignoring excess -Xlog options: \"%s\"", str);`

Reply:
Thank you for your suggestion. This message is clear for everyone. I changed the warning message.